apps/E2E/src/Button/specs/Button.spec.win.ts

```diff
@@ -31,17 +30,34 @@ describe('Button Accessibility Testing', () => {
   });
 
   it('Button - Validate accessibilityRole is correct', async () => {
```

Comment:
nit no need for "Button - " prefix (same for the next three tests). We already describe this as "Button Accessibility Testing".